datarequest.py

```diff
@@ -236,6 +238,26 @@ def transform_title(row):
                         ('items', colls)])
 
 
+@api.make()
+def api_schema_get(ctx, schema_name):
+    """Get a schema.
+
+       Arguments:
+       schema_name -- Name of schema.
+    """
+    # Define paths to schema and uischema
+    coll_path = "/" + user.zone(ctx) + "/" + SCHEMACOLLECTION
+    schema_path = coll_path + "/" + schema_name + "/" + "schema.json"
+    uischema_path = coll_path + "/" + schema_name + "/" + "uischema.json"
+
+    # Retrieve and read schema and uischema
```

Comment:
Check if file exists before reading and return proper API error if it does not exist.